sw/airborne/math/pprz_algebra_int.c

```diff
@@ -412,10 +412,10 @@ void int32_quat_of_axis_angle(struct Int32Quat *q, struct Int32Vect3 *uv, int32_
   PPRZ_ITRIG_SIN(san2, (angle / 2));
   int32_t can2;
   PPRZ_ITRIG_COS(can2, (angle / 2));
-  q->qi = can2;
-  q->qx = san2 * uv->x;
-  q->qy = san2 * uv->y;
-  q->qz = san2 * uv->z;
+  q->qi = (can2 << (INT32_QUAT_FRAC-INT32_TRIG_FRAC));
```

Comment:
Else where in this file, INT_MULT_RSHIFT is used to complete multiplications which multiplies then scales, any specific reason that isn't done here?

Reply:
it should be INT_MULT_LSHIFT eventually, but this one doesn't exist...